dags/notice_process_workflow.py

```diff
@@ -86,21 +86,25 @@ def _stop_processing():
     selector_branch_before_transformation = BranchPythonOperator(
         task_id=SELECTOR_BRANCH_BEFORE_TRANSFORMATION_TASK_ID,
         python_callable=_selector_branch_before_transformation,
+        trigger_rule=TriggerRule.ALL_DONE,
```

Comment:
What will be if one of upstream steps fail ?

Reply:
There are some problems with that. Next commit will fix this.